examples/test-upgrade/v090/cluster/v090/main.tf

```diff
@@ -9,7 +9,6 @@ resource "mongodbatlas_cluster" "test" {
   disk_size_gb                 = 100
   num_shards                   = 1
   replication_factor           = 3
-  provider_backup_enabled      = true
```

Comment:
why is this needed?

Reply:
this property is not expected in cluster, thrown by the [workflow here](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/6314926951/job/17146303344#step:11:2829)